packages/ui/feature-dashboard/src/lib/components/sidenav-routes-list/sidenav-routes-list.component.ts

```diff
@@ -6,36 +6,62 @@ import {
   AuthenticationService,
   EmbeddingService,
   NavigationService,
+  PlatformService,
+  UiCommonModule,
 } from '@activepieces/ui/common';
 import { Observable, forkJoin, map, of, take } from 'rxjs';
 import { ApFlagId, ProjectMemberRole, supportUrl } from '@activepieces/shared';
 import { DashboardService, FlagService } from '@activepieces/ui/common';
+import { SidenavRouteItemComponent } from '../sidenav-route-item/sidenav-route-item.component';
+import { CommonModule } from '@angular/common';
 
 type SideNavRoute = {
   icon: string;
   caption: string;
-  route: string;
+  route?: string;
```

Comment:
can you change this to `route: string | undefined`
instead of optional, so the at least we will be aware when it's not set in the route.